### PR TITLE
add logic to deploy to test

### PR DESCRIPTION
**Pull request by ranadewall**
I rolled this in from the existing deploy repo that was used when DCCS was an open source prov repo.

### CODE REVIEW

File: .github/workflows/deploy-test.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          token: ${{secrets.PAT}}
+          path: shared-actions
+
+      - name: Run ktlint
+        run: |
+          ./gradlew ktlint --parallel
+
+      - name: Build
+        run: |
+          ./gradlew --console=plain --refresh-dependencies clean build -x test
```
Comment (scirner22) [inline .github/workflows/deploy-test.yml:41]: testing is skipped?

**Review by scirner22 — DISMISSED** [review]

File: .github/workflows/deploy-test.yml (hunk 1 above)
Comment (ranadewall) [reply .github/workflows/deploy-test.yml]: good 👁️ thanks

**Comment by leeduan** [comment]
do we have concerns having these deploys in a PUBLIC repository? not sure we should add anything figure specific in open source

**Comment by ranadewall** [comment]
> do we have concerns having these deploys in a PUBLIC repository? not sure we should add anything figure specific in open source

I didn't realize when we moved it to figure land that it was still a public repo...

**Pull request closed without merging**